website/versioned_docs/version-0.10/commands.md

```diff
@@ -0,0 +1,52 @@
+---
+id: version-0.10-commands
+title: CLI commands
+original_id: commands
+---
+
+## Commands
+
+CLI offers help prompt. 
+Execute `graphback` in your shell for more information
+
+### Config 
+
+```bash
+graphback config
+```
+
+Create configuration in existing project to support Graphback resolver and schema generation.
+
+> **Note** When creating configuration please review if locations created for `files` integration match your project folder structure
+
+### Generate
+  ```bash
+  graphback generate
+  ```
+  Generate command will generate schema and resolvers for your GraphQL server based on your data model.
+
+### DB
+  ```bash
+  graphback db
+  ```
+  Create databases resources based on your data model. Currently supports PostgreSQL and sqlite3.
+> **Note** - *Running `graphback db` drops your database and recreates it with tables and relationships.*
+
+## Update Database
+
+```sh
+graphback update-db
+```
+
+Updates your database schema based on your data model. Currently supported change types are:
```

Comment:
This limitation is based on `migrateDBUsingSchema` which is not used by the CLI.